app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,104 @@
+class TeamsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create
+    company = Company.first
+    @team.company = company
+    if @team.save
+      flash[:success] = t('flash_messages.create', name: 'Team')
+    else
+      flash[:danger] = t('flash_messages.error', error_msg: @team.errors.full_messages.first)
+    end
+
+    respond_to do |format|
+      format.js { redirect_to teams_path }
+    end
+  end
+
+  def edit
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def update
+    if @team.update(team_params)
+      flash[:success] = t('flash_messages.update', name: 'Team')
+    else
+      flash[:danger] = t('flash_messages.error', error_msg: @team.errors.full_messages.first)
```

Comment:
Why first? Display all error messages. You may need to handle scenario of displaying flash messages from array in view then.